docs/pip/compatibility.md

```diff
@@ -462,3 +462,14 @@ is.
 For example, `uv pip install foo bar` prioritizes newer versions of `foo` over `bar` and could
 result in a different resolution than `uv pip install bar foo`. Similarly, this behavior applies to
 the ordering of requirements in input files for `uv pip compile`.
+
+## Installing a package from a specified directory
+
+UV does not reinstall already installed packages by default.
+
+Consequently, it will not behave like `pip install <folder>`, reinstalling every time.This is
```

Comment:
Missing space in `time.This`. :wink:

Reply:
![image](https://github.com/user-attachments/assets/6ab30d3b-b365-4243-8ea9-032684bf13bc)
Thank you, I think I got your point.